src/app.ts

```diff
@@ -89,6 +89,7 @@ export default class Client {
 
     this.root = root;
     this.root.id = "root";
+    conf.gameVersion = "2017.1.1.7";
```

Comment:
No problem, that makes more sense; let's just make sure we change it on every release, because I don't think our current config into app.ts is actually ever configured :confused: